### PR TITLE
[Term Entry] CSS Filter Function: Opacity() Example 2

### PR DESCRIPTION
**Pull request by giliopoulosaz**
### Description

- Added a image/media .png for opacity()
- Closes issue #3969 

### Type of Change

- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/css/concepts/filter-functions/terms/opacity/opacity.md [hunk 1]
```diff
@@ -38,3 +38,14 @@ Set the opacity of our element to `50%`:
...
+## Example 2
+
+Set the opacity of our element to `30%`:
+
+```css
+.banner-image {
+  filter: opacity(30%);
+}
+```
+![css-filterfunction-opacity-example2](https://raw.githubusercontent.com/Codecademy/docs/main/media/css-filterfunction-opacity-example2.png)
```
Comment (dakshdeepHERE) [inline content/css/concepts/filter-functions/terms/opacity/opacity.md:51]: @giliopoulosaz in the generated issue you were instructed to add media of the current example. No need to create a new example and add media for it.

**Review by dakshdeepHERE — COMMENTED** [review]
I have one comment for you @giliopoulosaz

**Comment by giliopoulosaz** [comment]
@dakshdeepHERE updated the file, removed Example 2 and just added media to the existing example.

**Review by dakshdeepHERE — APPROVED** [review]
Looks good to me @giliopoulosaz

File: content/css/concepts/filter-functions/terms/opacity/opacity.md [hunk 2]
```diff
@@ -38,3 +38,5 @@ Set the opacity of our element to `50%`:
   filter: opacity(50%);
 }
 ```
+
+![css-filterfunction-opacity-example](https://raw.githubusercontent.com/Codecademy/docs/main/media/css-filterfunction-opacity-example.png)
```
Comment (ishg-153) [inline content/css/concepts/filter-functions/terms/opacity/opacity.md:42]: Add a more descriptive `alt-text` (text inside the square brackets).

It should give information about the image.

Comment (ishg-153) [inline content/css/concepts/filter-functions/terms/opacity/opacity.md:42]: From your output remove the text below the images reading `Image with opacity filter` and `Image without opacity filter`.

```suggestion
![css-filterfunction-opacity-example](https://raw.githubusercontent.com/Codecademy/docs/main/media/css-filterfunction-opacity-example.png)

[add the description here]
```
Then add a sentence below the output that states which image is which.

Comment (ishg-153) [inline content/css/concepts/filter-functions/terms/opacity/opacity.md:42]: For the output itself, other entries show the original image first followed by the image with filter to the right. For the sake of uniformity, please reorder those images.

**Review by ishg-153 — CHANGES_REQUESTED** [review]
@giliopoulosaz I've suggested some changes. Please read all of them and then start making the changes.

**Comment by giliopoulosaz** [comment]
@ishg-153 made the changes you requested.

**Review by ishg-153 — APPROVED** [review]
@giliopoulosaz This looks ready to merge.

**Pull request merged**